cloudinit/config/cc_ansible.py

```diff
@@ -101,9 +104,15 @@ class AnsiblePullPip(AnsiblePull):
     def __init__(self, distro: Distro):
         self.cmd_pull = ["ansible-pull"]
         self.cmd_version = ["ansible-pull", "--version"]
-        self.env["PATH"] = ":".join([self.env["PATH"], "/root/.local/bin/"])
         self.distro = distro
 
+        self.env = os.environ.copy()
+        old_path = self.env.get("PATH")
+        if old_path:
```

Comment:
Is this right? If a PATH existed previous, add `/root/.local/bin` to it, but if not, set the PATH to empty? Why don't we want `/root/.local/bin/` in the else case?